tests/test_adjoint_jacobian.py

```diff
@@ -912,8 +904,14 @@ def circuit(params):
     qnode_def = qml.QNode(circuit, dev_def)
     qnode_lightning = qml.QNode(circuit, dev_lightning, diff_method="adjoint")
 
-    j_def = qml.jacobian(qnode_def)(params)
-    j_lightning = qml.jacobian(qnode_lightning)(params)
+    def casted_to_array_def(params):
+        return np.array(qnode_def(params))
+
+    def casted_to_array_lightning(params):
+        return np.array(qnode_lightning(params))
+
+    j_def = qml.jacobian(casted_to_array_def)(params)
+    j_lightning = qml.jacobian(casted_to_array_lightning)(params)
```

Comment:
As mentioned elsewhere, since this affects blogpost/forum code, just noting it may be worth annotating posts to reflect the change. This should also be added to the changelog as a breaking change for Lightning use.

Reply:
I've written a longer changelog entry.